setup.py

```diff
@@ -72,22 +72,28 @@
             "plone_addon = bobtemplates.plone.bobregistry:plone_addon",
             "plone_behavior = bobtemplates.plone.bobregistry:plone_behavior",
             "plone_buildout = bobtemplates.plone.bobregistry:plone_buildout",
-            "plone_content_type = bobtemplates.plone.bobregistry:plone_content_type",
+            "plone_content_type =" " bobtemplates.plone.bobregistry:plone_content_type",
```

Comment:
was this intended?

Reply:
?

Reply:
it looks like some lines were converted to 2 lines and then when converting back to one line, the quotes weren't removed.

I have removed them.